app-modules/application/src/Filament/Resources/ApplicationResource/Pages/ManageApplicationSubmissions.php

```diff
@@ -104,6 +119,15 @@ public function table(Table $table): Table
                 ViewAction::make()
                     ->modalHeading(fn (ApplicationSubmission $record) => "Submission Details: {$record->created_at}")
                     ->infolist(fn (ApplicationSubmission $record): array => [
+                        TextEntry::make('state')
+                            ->label('State')
```

Comment:
You don't actually need labels on a lot of these because the field name is the same.

Reply:
Fair, but imo it does not hurt to be specific.